src/Integration.Vsix.UnitTests/Analysis/DisableRuleCommandTests.cs

```diff
@@ -19,17 +19,15 @@
  */
```

Comment:
I have rmeoved all logic regarding the TryGetRuleId method and changed them to use the interface instead.